compiler/modules/print.wasm/tests/printBoolean.bal

```diff
@@ -0,0 +1,48 @@
+import ballerina/test;
+@test:Config{}
+l
+public function printBoolean() returns error? {
+
+    Module module = new;
+
+    Type[] localTypes = ["i32"];
+
+    module.addFunctionImport("println", "console", "log", "i32", "None");
+
+
+    // Get the arguement b
+    Expression b = module.localGet(0, "i32");
+
+    // Then call (b == true)
+    Expression[] thencallOperands = [makeConst(makeLiteralInt32(1))];
+    Expression thenCall = makeCall("println", thencallOperands, 1, "None");
```

Comment:
Relevant Binaryen C API is `BinaryenCall`, then shouldn't this be `call` ? where did the `make` come form?

Reply:
Yes in binaryen it is `BInaryenCall`.  I added `make` so that it would look I we are creating(making) a function call inside the `thenCall`. Thought that just having `call` there would look like invoking the function at that line in the code. Similarly in other place `makeConst`, `makeLiteralInt32`, `makeNop`, `makeReturn`.

Reply:
In nballerina repo we are following a succinct naming policy, and we are also interested in keeping in close to `Binaryen`, so I would go without `make` for all of those. If this was Java I would be Ok with longer names.